docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -195,25 +201,45 @@ AUTO will then query all available devices and remove CPU from the candidate lis
 
 Note that if you choose to exclude CPU from device candidate list, CPU will not be able to support the initial model compilation stage. See more information in [How AUTO Works](#how-auto-works).
 
-### Performance Hints for AUTO
-The `ov::hint::performance_mode` property enables you to specify a performance option for AUTO to be more efficient for particular use cases.
+### Checking Target Runtime Devices
 
-> **NOTE**: Currently, the `ov::hint` property is supported by CPU and GPU devices only.
+To query the runtime target devices on which the inferences are being executed using AUTO, you can use the `ov::execution_devices` property. It must be used with `get_property`, for example:
 
-#### THROUGHPUT
-This option prioritizes high throughput, balancing between latency and power. It is best suited for tasks involving multiple jobs, such as inference of video feeds or large numbers of images.
+@sphinxdirective
 
-> **NOTE**: If no performance hint is set explicitly, AUTO will set THROUGHPUT for devices that have not set `ov::device::properties`. For example, if you have both a CPU and a GPU in the system, this command `core.compile_model("AUTO", ov::device::properties("CPU", ov::enable_profiling(true)))` will set THROUGHPUT for the GPU only. No hint will be set for the CPU although it's the selected device.
+.. tab:: C++
+
+   .. code-block:: sh
+
+      ov::Any execution_devices = compiled_model.get_property(ov::execution_devices)
+
+.. tab:: Python
+
+   .. code-block:: sh
+
+      execution_devices = compiled_model.get_property("EXECUTION_DEVICES")
```

Comment:
put code snippets to `docs/snippets`

Reply:
The example code has been moved to `docs/snippets`